jsonpb/jsonpb_test.go

```diff
@@ -553,12 +553,12 @@ var unmarshalingTests = []struct {
 	{"camelName input", Unmarshaler{}, `{"oBool":true}`, &pb.Simple{OBool: proto.Bool(true)}},
 
 	{"Duration", Unmarshaler{}, `{"dur":"3.000s"}`, &pb.KnownTypes{Dur: &durpb.Duration{Seconds: 3}}},
-	{"null Duration", Unmarshaler{}, `{"dur":null}`, &pb.KnownTypes{Dur: &durpb.Duration{Seconds: 0}}},
+	{"null Duration", Unmarshaler{}, `{"dur":null}`, &pb.KnownTypes{Dur: nil}},
```

Comment:
Personally, I think the intent is clear, i.e. the field is not being set.  Also, I fear that some linter tool may flag/change this to simplify the notation.  I've seen some PRs to simplify code.

Anyways, I don't have a strong opinion on this and am ok if you think it should be there.  Something that I wish to do if I get a chance is to clean up the structure of these tests, i.e. it'll be good to specify field names for all these test cases and hence also have more line breaks to make it more readable, e.g.
{
    desc: "null Duration",
    unmarshaler: ...,
    ...
},